bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -77,7 +74,8 @@
       new Predicate<MutateRowRequest>() {
         @Override
         public boolean apply(MutateRowRequest mutateRowRequest) {
-          return allCellsHaveTimestamps(mutateRowRequest.getMutationsList());
+          return mutateRowRequest != null
```

Comment:
The test cases for mutate and checkAndMutate pass in null.  We don't have equivalent tests for the mutateRows predicate.  I'll add it.